test/components/Menu.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+// import renderer from 'react-test-renderer';
```

Comment:
if I don't comment this out, the tests fail. Importing `enzyme` and `react-test-renderer` together doesn't work because of a bug in React that's already fixed in master and will land in react 15.4